src/index.html

```diff
@@ -52,10 +53,18 @@
   </div>
  
 </header>
+
+<main class="p-5">
+  <section id="mainContainer" class="main-container">
+    
```

Comment:
I think it may have been better if you use only one selector, either `id="mainContainer"` or `class="main-container"`. Can you please consider using one of them?

Reply:
Thanks for pointing it out. I implemented the changes you suggested.